### PR TITLE
Adopt the Contributor Covenant

### PR DESCRIPTION
**Pull request by notriddle**

### CODE REVIEW

**Comment by notriddle** [comment]
🤦‍♂️ wrong issue number.

**Comment by Macarse** [comment]
👍

**Comment by Macarse** [comment]
@notriddle what's blocking this PR?

**Comment by notriddle** [comment]
Nothing, really. I just want to give people a chance to weigh in. I'm not really an expert on this kind of thing...

**Comment by Macarse** [comment]
@notriddle oh, I thought you copied and pasted it from somewhere else :)

**Comment by notriddle** [comment]
It is basically the same, except for the stuff in the second commit. I need to get that email address set up. 😄

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+Examples of project spaces include all repositories in the bors-ng GitHub
+organization, the `#bors` channel on the Mozilla IRC network, the blog,
+and any bors meetup.
+
+CONNECTING A GITHUB REPOSITORY TO THE PUBLIC BORS-NG INSTANCE DOES NOT MAKE IT
+A BORS-NG PROJECT SPACE. The public instance is governed by the
+[bors.io terms of service], a document with a narrower focus than this one.
```
Comment (matklad) [inline CODE_OF_CONDUCT.md]: The link is missing?

**Comment by notriddle** [comment]
Status update: waiting on CloudFlare to approve the SSL key for bors.tech. Once that's done, I'll be able to set up an email forward for community@bors.tech, which will allow the CoC to be put into production.

**Comment by notriddle** [comment]
I just tested the mailing list; it works.

**Pull request merged**